cli/integration_tests/basic_monorepo/dry_run.t

```diff
@@ -0,0 +1,97 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd)
+
+Check my-app#build output
+  $ ${TURBO} run build --dry | grep "Packages in Scope" -A 4
+  Packages in Scope
+  Name   Path          
+  my-app apps/my-app   
+  util   packages/util 
+  
+  $ ${TURBO} run build --dry | grep "my-app#build" -A 12
+  my-app#build
+    Task                   = build                                                                                             
+    Package                = my-app                                                                                            
+    Hash                   = 8ee3ee4bb0bb88be                                                                                  
+    Cached (Local)         = false                                                                                             
+    Cached (Remote)        = false                                                                                             
+    Directory              = apps/my-app                                                                                       
+    Command                = echo 'building'                                                                                   
+    Outputs                =                                                                                                   
+    Log File               = apps/my-app/.turbo/turbo-build.log                                                                
+    Dependencies           =                                                                                                   
+    Dependendents          =                                                                                                   
+    ResolvedTaskDefinition = {"outputs":[],"cache":true,"dependsOn":[],"inputs":[],"outputMode":0,"env":[],"persistent":false} 
```

Comment:
I thought about _not_ printing json here and trying have nested lines, but I think we want json, since it's trying to reflect user's input.